src/ResourceManager/Compute/Commands.Compute/Extension/VmssDiskEncryption/SetAzureVmssDiskEncryptionExtension.cs

```diff
@@ -92,7 +92,10 @@ public class SetAzureVmssDiskEncryptionExtensionCommand : VirtualMachineScaleSet
             Mandatory = false,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Type of the volume (OS or Data) to perform encryption operation")]
-        [ValidateSet("OS", "Data", "All")]
+        [ValidateSet(
+            AzureDiskEncryptionExtensionContext.VolumeTypeOS,
```

Comment:
Will this break / change the existing customer scripts ?

Reply:
These are constants with the same string values as the validateset above, which should have no impact:https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/AzureDiskEncryptionExtensionContext.cs#L33